.github/workflows/release.yaml

```
@@ -1,72 +1,42 @@
-      'workflow_dispatch' && startsWith(github.event.inputs.ref_name, 'v'))
-    needs:
-      - skaffold
-      - nix
+    if: |
+      (startsWith(github.ref, 'refs/tags/v')) ||
+      (github.event_name == 'workflow_dispatch' && startsWith(github.event.inputs.ref_name, 'v'))
+    runs-on: ubuntu-slim
     permissions:
       contents: write
-    runs-on: ubuntu-slim
+      workflows: write
```

actionlint (1.7.12)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)